api/src/test/resources/application.properties

```diff
@@ -7,6 +7,14 @@ spring.jpa.properties.hibernate.dialect=org.pmiops.workbench.cdr.CommonTestDiale
 spring.liquibase.enabled=false
 # To allow each tests override and customize their the beans.
 spring.main.allow-bean-definition-overriding=true
+
+# Avoid log spam in tests
+spring.main.log-startup-info=OFF
+spring.main.banner-mode=off
+logging.level.org.springframework=ERROR
+spring.jpa.properties.hibernate.show_sql=false
```

Comment:
Fixed duplication. From trial and error, `show_sql` is not off by default. If I comment this out, I get tons of logspam from Hibernate in tests.

Reply:
It is turned on here: https://github.com/all-of-us/workbench/blob/master/api/src/test/java/org/pmiops/workbench/testconfig/TestJpaConfig.java#L66

Reply:
I found this as well yesterday and tried removing that line, it didn't make a difference. Note: that config is only imported into certain tests. I suppose I can remove this as part of the PR as well though for consistency. This may still affect the tests where it does get imported.